docs/image-variants.md

```diff
@@ -0,0 +1,40 @@
+# Image variants
+
+The Flex API allows requesting custom image variants by accompanying an image
+variant sparse attribute parameter with a corresponding image variant
+definition:
```

Comment:
This is a dense sentence with important details. Could you at link to the reference for sparse attributes, here or in the end of the doc?